__tests__/app.test.js

```diff
@@ -177,7 +179,16 @@ describe("GET /api/users", () => {
       });
   });
 });
-// Responds with:
 
-// an array of objects, each object should have the following property:
-// username
+exports.selectArticles = () => {
```

Comment:
Can you please move this in the correct models file? You've put it in your test file 😅

Reply:
I did now, I think I have been playing with this function. Early morning for me :)

Reply:
It was in models file, actually, but I got it copied to the test I remember now so I can compare with what I had before, as well . But it sits on the models now only. Tests are working